packages/atlas/src/components/_video/VideoPlayer/videoJsPlayer.ts

```diff
@@ -2,15 +2,17 @@ import { RefObject, useEffect, useRef, useState } from 'react'
 import { VideoJsPlayer, VideoJsPlayerOptions } from 'video.js'
 import videojs from 'video.js/dist/alt/video.core.novtt'
 
+import { useGetAssetUrl } from '@/hooks/useGetAssetUrl'
+
 export type VideoJsConfig = {
-  src?: string | null
+  src?: string[] | null
```

Comment:
I'm not sure here, but will be good to rename to smth like `srcArr`/`srcList`/`srcs`/
@drillprop @WRadoslaw or maybe you know the better naming?